composer/trainer/trainer.py

```diff
@@ -868,6 +868,12 @@ def __init__(
         self.state.evaluators = evaluators
 
         # Grad Clip Norm
+        if grad_clip_norm > 0:
+            warnings.warn(
+                DeprecationWarning(
+                    (f"Using the 'grad_clip_norm' field in Trainer is deprecated. PLease use"
+                     "the GradientClipping Algorithm in composer/algorithms/gradient_clipping")))
+        
         self._grad_clip_norm = grad_clip_norm
```

Comment:
Since it's an alg now, can we remove `self._grad_clip_norm` from the trainer?

Reply:
removed everywhere except for used once in `__init__`